test/unit/spec/DeviceFingerprint_spec.js

```diff
@@ -0,0 +1,73 @@
+/*global JSON */
+define([
+  'jquery',
+  'vendor/lib/q',
+  'helpers/util/Expect',
+  'sandbox',
+  'util/DeviceFingerprint'
+],
+function ($, Q, Expect, $sandbox, DeviceFingerprint) {
+
+  Expect.describe('DeviceFingerprint', function () {
+
+    function mockIFrameMessages(success, errorMessage) {
+      var message = success ? {
+        type: 'FingerprintAvailable',
+        fingerprint: 'thisIsTheFingerprint'
+      } : errorMessage;
+      window.postMessage(JSON.stringify(message), '*');
+    }
+
+    beforeEach(function () {
+      $sandbox.append('<div class="auth-content"></div>');
+    });
+
+    it('iframe is created with the right src and it is hidden', function () {
+      spyOn(window, 'addEventListener');
+      DeviceFingerprint.generateDeviceFingerprint('baseUrl');
+      var $iFrame = $('#okta_fingerprint_iframe');
```

Comment:
Do we still need the ID? I.e. in this case, you just need to check the $sandbox for an iframe right?